docs/build/isc/v1.0.0-rc.6/docs/how-tos/create-foundry.md

```diff
@@ -0,0 +1,83 @@
+---
+description: How to create a L1 foundry
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - foundry
+  - EVM
+  - how-to
+---
+# Create a Foundry
+## About Foundries
+
+The Stardust update allows you to create your own native tokens. Native tokens are minted by a [Foundry](/tips/tips/TIP-0018/#foundry-output). 
+The Foundry allows you to specify your native token's maximum supply **once** and change the circulating supply. 
+This guide will show you how to create an L1 foundry using a L2 smart contract.
+
+## Example Code
+
+:::info Ownership
+
+You might want to look into making the function ownable with, for example,
+[OpenZeppelin](https://docs.openzeppelin.com/contracts/5.x/access-control#ownership-and-ownable)
+so only owners of the contract can call certain functionalities of your contract.
+
+:::
+
+1. Check if the amount paid to the contract is the same as the required [storage deposit](/learn/protocols/stardust/core-concepts/storage-deposit) 
+   and set the allowance.
+    
+    ```solidity
+            require(msg.value == _storageDeposit*(10**12), "Please send exact funds to pay for storage deposit");
+            ISCAssets memory allowance;
+            allowance.baseTokens = _storageDeposit;
+    ```
+
+:::info  Payable
+
+Instead of making the function payable, you could let the contract pay for the storage deposit. 
+If so, you will need to change the `require` statement to check if the contract's balance has enough funds.
```

Comment:
It'd be good to provide an alternative line if the users want to use it.

Reply:
I can add that in one of the other PRs where the admonition got exported 😆